borgmatic/config/schema.yaml

```diff
@@ -29,19 +29,40 @@ properties:
             repositories:
                 type: array
                 items:
-                    type: string
-                description: |
-                    Paths to local or remote repositories (required). Tildes are
-                    expanded. Multiple repositories are backed up to in
-                    sequence. Borg placeholders can be used. See the output of
-                    "borg help placeholders" for details. See ssh_command for
-                    SSH options like identity file or port. If systemd service
-                    is used, then add local repository paths in the systemd
-                    service file to the ReadWritePaths list.
+                    type: object 
+                    required:
+                        - path
+                    properties:
+                        path:
+                            type: string
+                            description: |
+                                Path to local or remote repository
+                                (required). Tildes are expanded.
+                                Multiple repositories are backed up
+                                to in sequence. Borg placeholders
+                                can be used. See the output of
+                                "borg help placeholders" for
+                                details. See ssh_command for SSH
+                                options like identity file or port.
+                                If systemd service is used, then
+                                add local repository paths in the
+                                systemd service file to the
+                                ReadWritePaths list.
+                            example: ssh://user@backupserver/./{fqdn}
+                        label:
+                            type: string
+                            description: |
+                                Optional label for the repository. This 
+                                can be used with the --repository option 
```

Comment:
"option" -> "flag" here. In user-facing stuff, I try to be pretty rigorous about using "option" to refer to configuration file values and "flag" to refer to command-line arguments. Cuts down on confusion, IMO.